cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala

```diff
@@ -104,26 +145,28 @@ class SpatialFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
-    section("point() - WGS 84", "functions-point") {
-      p("`point()` returns a point in the _WGS 84_ coordinate system corresponding to the given coordinate values.")
-      function("point({longitude | x, latitude | y [, crs]})", "A Point.", ("A single map consisting of the following:", ""), ("longitude/x", "A numeric expression"), ("latitude/y", "A numeric expression"), ("crs", "The string 'WGS-84'"))
+    section("point() - WGS 84 2D", "functions-point-wgs84-2d") {
+      p("`point(longitude|x, latitude|y)` returns a 2D point in the _WGS 84_ CRS category -- i.e. a point in the WGS 84 2D CRS -- corresponding to the given coordinate values.")
```

Comment:
This is a bit clunky--the same sentence is used to explain what the function returns, and that a 2D point in the WGS84 category is the same as a point in the WGS84 2D CRS.

Could it just say `returns a point in the WGS 84 2D CRS corresponding to the given coordinate values.` ?

Avoid "e.g." and "i.e." where possible, if it has to be used then add the comman: "i.e." => "i.e.,"